weave/trace_server/clickhouse_trace_server_batched.py

```diff
@@ -904,6 +921,52 @@ def file_content_read(self, req: tsi.FileContentReadReq) -> tsi.FileContentReadR
             raise ValueError("Missing chunks")
         return tsi.FileContentReadRes(content=b"".join(chunks))
 
+    def feedback_create(
+        self, req: tsi.FeedbackCreateReqForInsert
+    ) -> tsi.FeedbackCreateRes:
+        feedback_id = generate_id()
+        created_at = datetime.datetime.now(ZoneInfo("UTC"))
+        # TODO: Any validation on weave_ref?
+        # TODO: What should max payload size be?
```

Comment:
1024 seems like a pretty sufficient cap for now - couple paragraphs in note form

Reply:
I picked a number rather arbitrarily - small because it is easier to loosen than tighten, but I'm open to alternatives.